src/util/Util.js

```diff
@@ -470,7 +470,7 @@ class Util extends null {
     }
 
     if (color < 0 || color > 0xffffff) throw new RangeError('COLOR_RANGE');
-    else if (color && isNaN(color)) throw new TypeError('COLOR_CONVERT');
+    else if (isNaN(color)) throw new TypeError('COLOR_CONVERT');
```

Comment:
Just so `color` isn't casted to a number (even when it's a string):
```suggestion
    else if (Number.isNaN(color)) throw new TypeError('COLOR_CONVERT');
```